### ISSUE TITLE
Training mode usage and documentation

### ISSUE THREAD

**Issue by kyleLesack** [timothyjamesbecker/FusorSV#17]
Hello,

I would like to use SVE/FusorSV for calling variants in _C. elegans_. I have generated mock genomes with simulated structural variants that I would like to use as training data, as I don't want to use a model trained on human data. 

Could you please point me towards the documentation for using the training mode or show me how to generate new models (the SVE documentation mentions appending to models, which would also be helpful)?

Thanks

**Comment by timothyjamesbecker** [comment]
Training mode is conducted by not using a fusion model.  This is achieved by not using the -f argument and supplying a model file.  To train your own model, provide an input that has for every sample folder a matching * _S0.vcf file to use as the truth.  Take a look at the example data set used in the paper which is under releases (first one).  Before you try training your own, try to train the default model by this data set first since it is formatted correctly.
